frontend/components/UserSettings.vue

```diff
@@ -0,0 +1,978 @@
+                        </button>
+                        <SettingsEmailChange
+                            :visible="isEmailChangeDialogVisible"
+                            :curr-email="currUser.email"
+                            @close="isEmailChangeDialogVisible = false"
+                            @change-email="changeEmail"
+                        />
+                    </div>
+                </div>
+                <div class="flex pb-5 pl-4">
+                    <div class="mr-5 flex w-[50%] flex-col">
+                        <div class="text-xl">
+                            <T key-name="dashboard.user.settings.password" />
+                        </div>
+                        <div class="text-sm text-text dark:text-natural-300">
+                            <T
+                                key-name="dashboard.user.settings.password.description"
+                            />
+                        </div>
+                    </div>
+                    <div class="flex w-[45%] items-center justify-end">
+                        <button
+                            class="mr-4 flex w-32 justify-center rounded-md border-2 border-dandelion-300 bg-natural-50 px-2 py-0.5 text-base font-medium hover:bg-dandelion-200 dark:border-dandelion-300 dark:bg-natural-900 dark:hover:bg-pesto-600"
+                            @click="
+                                isPasswordChangeDialogVisible =
+                                    !isPasswordChangeDialogVisible
+                            "
+                        >
+                            <T key-name="common.change" />
+                            <span class="pi pi-angle-right self-center" />
+                        </button>
+                        <SettingsPasswordChange
+                            :visible="isPasswordChangeDialogVisible"
+                            @close="isPasswordChangeDialogVisible = false"
+                        />
+                    </div>
+                </div>
+                <div class="flex items-center pb-2">
+                    <div class="h-0.5 w-6 bg-calypso-400" />
+                    <div class="flex-grow-5 mx-2 text-2xl font-semibold">
+                        <T key-name="dashboard.user.settings.appearance" />
+                    </div>
+                    <div class="h-0.5 flex-grow bg-calypso-400" />
+                </div>
+                <div class="flex pb-5 pl-4">
+                    <div class="flex w-full flex-col">
+                        <div class="text-xl">
+                            <T key-name="dashboard.user.settings.appearance" />
+                        </div>
+                        <div class="text-sm text-text dark:text-natural-300">
+                            <T
+                                key-name="dashboard.user.settings.appearance.description"
+                            />
+                        </div>
+                    </div>
```

Comment:
![image](https://github.com/user-attachments/assets/441f1be3-1478-46fc-ac27-9e5eb3982b92)
can u allign the dropdown menu with the buttons